serverless/src/api/municipalities/createMunicipalitiesStats/index.js

```diff
@@ -144,10 +149,10 @@ const computeStats = async ({
   newcomers = newcomers.slice(0, amountOfNewcomers);
 
   relativeChangers.sort((a, b) => a.relativeChange - b.relativeChange);
-  relativeChangers = relativeChangers.slice(0, amountOfChanged);
+  relativeChangers = relativeChangers.slice(0, amountOfChanged / 2);
```

Comment:
Why / 2 ?

Reply:
Sah in der Animation nicht gut aus, wenn in der gesamten change-Kategorie so viele Gemeinden sind. Da es zwei Unterkategorien gibt, hab ich da durch zwei geteilt, so dass die relativen und absoluten Veränderungen dann amountOfChanged ergibt.